cpp/test/neighbors/ann_cagra.cuh

```
@@ -497,9 +497,13 @@ class AnnCagraFilterTest : public ::testing::TestWithParam<AnnCagraInputs> {
         search_params.algo         = ps.algo;
         search_params.max_queries  = ps.max_queries;
         search_params.team_size    = ps.team_size;
-        search_params.itopk_size   = ps.itopk_size;
         search_params.hashmap_mode = cagra::hash_mode::HASH;
 
+        // TODO: setting search_params.itopk_size here breaks the filter tests, but is required for
+        // k>1024 skip these tests until fixed
+        if (ps.k >= 1024) return;
```

Make gtest mark the current case as skipped rather than silently accepting it.
```suggestion
        if (ps.k >= 1024) { GTEST_SKIP(); }
```